tests/test_enforce_once_required_parameter.py

```diff
@@ -101,7 +101,7 @@ def test_schema():
 
 
 def test_get_invalid():
-    response = client.get("/foo", params={"client_id": None})
```

Comment:
The new client has different behavior. Setting a param _A_ explicitly to `None` will send the param _A_ with an empty string value, for example
```python
QueryParams('client_id=')
```